Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -74,13 +77,30 @@ private void IgniteOnCollide(EntityUid uid, IgniteOnCollideComponent component,
             Ignite(otherFixture, flammable);
         }
 
+        private void OnMapInit(EntityUid uid, FlammableComponent component, MapInitEvent args)
+        {
+            // Sets up a fixture for flammable collisions.
+            // TODO: Should this be generalized into a general non-hard 'effects' fixture or something? I can't think of other use cases for it.
+            // This doesn't seem great either (lots more collisions generated) but there isn't a better way to solve it either that I can think of.
```

Comment:
area of note

Reply:
could add groups specifically for this so that they only collide with the other flammable fixtures and don't generate other collisions but like. that seems a lot ickier